### PR TITLE
TRACING-3955 Documentation for using openshift service CA in Tempo

### PR DESCRIPTION
**Pull request by rubenvp8510**
Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by max-cx** [comment]
@rubenvp8510, thank you.
I continue working on this content in https://github.com/openshift/openshift-docs/pull/79813.
You are welcome to close this PR.

File: modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+----
+
+<1> Must be set to `true` for enable TLS on the distributor
+<2> The secret that contains the tls.key and tls.crt files.
+<3> Optionally a CA if you want to enable mTLS
+
+In OpenShift you can also use the Service serving certificates, if you want to use it you only need to enabled TLS, no need to specify certificate secret. If you specify one, that will be use, otherwise OpenShift service certificates are used, this mode doesn't support mTLS at the moment.
```
Comment (max-cx) [inline modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc:44]: By "Service serving certificates", do you mean
this
https://docs.openshift.com/container-platform/4.14/security/certificate_types_descriptions/service-ca-certificates.html#cert-types-service-ca-certificates
and also this
https://docs.openshift.com/container-platform/4.14/security/certificates/service-serving-certificate.html
?

File: modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-receiver-tls_{context}"]
+= Configure receiver TLS
+
+The `TempoMonolithic` custom resource (CR) allow to configure TLS on the distributor receivers when is not used with the gateway. You can specify a certificate, first you need to create a Secret with the tls.key and tls.crt keys, Those keys will contains the certificate and the key. Optionally you can specify the CA in a ConfigMap if you want to enable mutual TLS authentication.
```
Comment (max-cx) [inline modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc:5]: By "You can specify a certificate" do you mean
this
https://docs.openshift.com/container-platform/4.14/security/certificate_types_descriptions/user-provided-certificates-for-default-ingress.html#cert-types-user-provided-certificates-for-default-ingress
or
this
https://docs.openshift.com/container-platform/4.14/security/certificate_types_descriptions/user-provided-certificates-for-api-server.html#cert-types-user-provided-certificates-for-the-api-server
?

Comment (rubenvp8510) [reply modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc:5]: None of those I think, we have specified fields on the CRD to set the configmap and secrets for the custom certificates.

File: modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc (hunk 1 above)
Comment (rubenvp8510) [reply modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc:44]: Yes,

**Pull request closed without merging**